Makefile

```diff
@@ -75,6 +76,9 @@ lint-frontend:
 shell:
 	"${DC}" run --rm webapp //bin/bash
 
+sync-projects:
+	"${DC}" run --rm webapp .//manage.py sync_projects $(opts)
+
```

Comment:
Let's keep the order of items here the same as below.

And let's also document the missing targets: `flake8` and `test-frontend`.